lib/MusicBrainz/Server/WebService/Serializer/JSON/2/Release.pm

```diff
@@ -87,8 +87,8 @@ sub serialize
         # any), those of the release group should be serialized (to match the
         # behaviour of the XML serializer). So we call `serialize_rating`
```

Comment:
The comment here ought to be updated too.

Reply:
hmm yes, and we can skip the intermediate $output below